### PR TITLE
Vitess Slack Guidelines v1.0

### PR DESCRIPTION
**Pull request by askdba**
Signed-off-by: Alkin Tezuysal <alkin.tezuysal@gmail.com>

### CODE REVIEW

File: doc/VitessSlackGuidelinesv1_0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+## Channel Spamming
+Slack is our best communication medium for the community. Please keep your messages on topic for each channel, and please use the appropriate channel. Spamming channels with unrelated or marketing messages is not allowed.
+
+## Unwanted Solitication through Private Messages
+Directly messaging users (unless the user has specifically requested the information) offering services, events, employment opportunities or freelance projects, is not allowed.
+
+### Unwanted Voice/Video Calling
+Abuse of the voice/video calling features to call users without their permission is not allowed. Accidents sometimes happen, let the user know it was a mistake.
+
+## Reporting a Problem, Issue ,or Bug
```
Comment (deepthi) [inline doc/VitessSlackGuidelinesv1_0.md:14]:
```suggestion
## Reporting a Problem, Issue, or Bug
```

**Review by deepthi — COMMENTED** [review]
Should we add attribution? That this is adapted from https://github.com/cncf/foundation/blob/master/slack-guidelines.md?

File: doc/VitessSlackGuidelinesv1_0.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Vitess Slack Guidelines v1.0
+
+The [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md) applies in all areas in addition to these rules:
+
+## Channel Spamming
+Slack is our best communication medium for the community. Please keep your messages on topic for each channel, and please use the appropriate channel. Spamming channels with unrelated or marketing messages is not allowed.
+
+## Unwanted Solitication through Private Messages
```
Comment (rohit-nayak-ps) [inline doc/VitessSlackGuidelinesv1_0.md:8]: typo: Solitication => Solicitation

**Comment by systay** [comment]
This feels more like website content and maybe not so much something that belongs here, but I'm not sure. What was the thinking for adding it here, where it probably won't be seen much?

**Comment by deepthi** [comment]
> This feels more like website content and maybe not so much something that belongs here, but I'm not sure. What was the thinking for adding it here, where it probably won't be seen much?

Once it is merged we'll publish it on vitess.slack.com and link to it in the #general channel.
Going forward if there are any violations of slack etiquette we should be able to point to the written policy.

**Review by shlomi-noach — APPROVED** [review]

**Comment by shlomi-noach** [comment]
Approved, but like @systay I feel like this belong in a different repo than `vitessio/vitess`.

**Comment by askdba** [comment]
> Approved, but like @systay I feel like this belongs in a different repo than `vitessio/vitess`.

All Vitess documentation except the Website is in vitessio/vitess repo as far as I know.

**Review by rohit-nayak-ps — APPROVED** [review]
lgtm

**Comment by shlomi-noach** [comment]
> All Vitess documentation except the Website is in vitessio/vitess repo as far as I know.

Fair. This makes sense.

**Comment by askdba** [comment]
@deepthi should we as a non-PS maintainer to review this?

**Comment by sougou** [comment]
> @deepthi should we ask a non-PS maintainer to review this?

Not necessary, because the content originates from CNCF.

> > All Vitess documentation except the Website is in vitessio/vitess repo as far as I know.
> 
> Fair. This makes sense.

There's probably a better place to put this, but this is a good start. We can move it later as we reorganize things.

**Pull request merged**